tests/unit/matrix/test_assembler.py

```diff
@@ -1561,11 +1562,26 @@ def it_provides_the_order(self, request):
             request, _SortRowsByMarginalHelper, "_empty_row_idxs"
         )
 
-        assert _SortRowsByMarginalHelper(None, None)._order
+        _SortRowsByMarginalHelper(None, None)._order
 
         SortByValueCollator_.display_order.assert_called_once_with(
             _rows_dimension_(),
             _element_values_(),
             _subtotal_values_(),
             _empty_row_idxs_(),
         )
+
+    def it_provides_the_element_values_to_help(self, request, _marginal_prop_):
```

Comment:
`request` is not necessary unless it is used (easy to forget on copy-paste, I do it frequently :).